server/router/router.go

```diff
@@ -188,6 +190,29 @@ func (router *router) HandleMessage(message *protocol.Message) error {
 	return nil
 }
 
+func (router *router) GetSubscribersForTopic(topicPath string) (JSONResponse []byte, err error) {
+
+	sliceOfRouteParams := make([]map[string]string, 0)
+
+	slice, present := router.routes[protocol.Path(topicPath)]
+	if present {
+		for index, currRoute := range slice {
+			logger.WithFields(log.Fields{
+				"index":      index,
+				"routeParam": currRoute.RouteParams,
+			}).Info("Added route to slice")
+			sliceOfRouteParams = append(sliceOfRouteParams, currRoute.RouteParams)
+		}
+		serialized, err := json.Marshal(sliceOfRouteParams)
+		logger.WithField("json_Response", string(serialized)).Info("Serialized json as string")
+		if err != nil {
+			return nil, err
+		}
+		return serialized, nil
+	}
+	return nil, ErrInvalidRoute
```

Comment:
Why return invalid route in this case. If topic is not found better return an empty data.
Don't think it's needed to error. Maybe no one subscribed in that topic which makes us ok with that.

This method should always return an empty serialized slice.